cax/cax.json

```diff
@@ -69,10 +71,15 @@
     "receive": "scp",
     "hostname": "tegner-login-1.pdc.kth.se",
     "directory": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_raw": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_root": "/cfs/klemming/projects/xenon/xenon1t/processed",
     "upload_options": [],
     "username": "bobau",
     "download_options": [
       "midway-login1"
+    ],
+    "pax_versions": [
+      "v4.10.0"
```

Comment:
I see the problem. It is attempting to use this json definition. General question: Do we ever want situation such:
"pax_versions": ["v4.9.5", "v4.10.0"]
Where we reprocess data with two different pax versions at the same time?
If the answer is yes then I can not use this definition any more and I have to define another one in addition:
(e.g.) "pax_root": = "v4.10.0"